ledger-core/v2/runner/executor/builtin/builtin.go

```diff
@@ -94,21 +95,26 @@ func (r *Runner) CallMethod(
 		return nil, nil, errors.New("failed to find contracts method")
 	}
 
-	if methodObject.Unordered != callCtx.Unordered {
-		orderedUnordered := func(unordered bool) string {
-			if unordered {
-				return "Unordered"
-			}
-			return "Ordered"
-		}
-
-		return nil, nil, errors.Errorf("calling %s method as %s",
-			orderedUnordered(methodObject.Unordered),
-			orderedUnordered(callCtx.Unordered),
-		)
+	return methodObject.Func(data, args)
+}
+
+func (r *Runner) ClassifyMethod(_ context.Context,
+	codeRef reference.Global,
+	method string) (contract.MethodIsolation, error) {
+
+	contractName, ok := r.CodeRefRegistry[codeRef]
+	if !ok {
+		return contract.MethodIsolation{},
```

Comment:
Isn't it better to split that return to 2 step?
```
err := throw.E("failed to find contract with reference", struct{ Reference reference.Global }{Reference: codeRef})
return contract.MethodIsolation{}, err
```

Reply:
fixed